src/Components/Indicators/PageIndicator/PageIndicator.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import "./PageIndicator.scss";
+
```

Comment:
Why you dont named this Page or pageNumber?

Reply:
Because it is an indicator. The others are called indicators too

![sdfsdfs](https://user-images.githubusercontent.com/54618409/119740814-d85f0b00-be84-11eb-978b-1d2a5f78fe18.PNG)

Reply:
Sorry, I clicked the wrong line on my cell phone. Correctly, I meant in the props.

Reply:
Ah ok. Done